### DISCUSSION TITLE
Ghost notes?

### DISCUSSION THREAD

**Discussion by leadpoisoned07** [ryohey/signal#377] [Q&A]
Hi, I have been using Signal for a while now. However, I keep seemingly randomly run into this issue where when I place notes, they have a very low volume and are black on the inside, unlike the standard blue. Is this possibly a ghost note feature? Is there a way to revert back to normal notes other than refreshing the page? Thanks in regard!
![Schermafbeelding 2024-06-27 080323](https://github.com/ryohey/signal/assets/77795640/0b82c3ad-6dfe-4e0d-9bb8-074ffddc8bea)

**Comment by ryohey** [comment]
Try dragging the blue bar in the velocity pane to a higher value. 
<img width="259" alt="image" src="https://github.com/ryohey/signal/assets/5355966/953090d0-5357-4190-942d-9ed871eafc81">

**Reply by leadpoisoned07** [reply]
Ah, I see that velocity impacts the opacity of the note. I still wonder why it does it randomly though.
